src/gsi/m_berror_stats.f90

```diff
@@ -869,7 +1096,7 @@ subroutine setcorchem_(cname,corchem,rc)
          enddo
       enddo
    enddo
-   work_chem(nsig+1,mm1)=real(lon1*lat1,r_kind)
+   work_chem(nsig+1,mm1)=float(lon1*lat1)
```

Comment:
same question as above

Reply:
Revert back to `develop` to use `real` and `r_kind`.  Make the same change on line 865.  Done at 68d52ac.